x-pack/qa/multi-project/core-rest-tests-with-multiple-projects/build.gradle

```diff
@@ -61,7 +63,7 @@ tasks.named("yamlRestTest").configure {
     '^snapshot.create/*/*',
     '^snapshot.delete/*/*',
     '^snapshot.get/*/*',
-    '^snapshot.get_repository/20_repository_uuid/*',
+    '^snapshot.get_repository/*/*',
```

Comment:
Same mute here. They will get unmute again once repositoriy and snapshot are truly MP ready.